eio_main.opam

```diff
@@ -10,11 +10,13 @@ doc: "https://ocaml-multicore.github.io/eio/"
 bug-reports: "https://github.com/ocaml-multicore/eio/issues"
 depends: [
   "dune" {>= "3.0"}
-  "eio_linux" {= version & os = "linux"}
   "mdx" {>= "1.10.0" & with-test}
-  "eio_luv" {= version}
+  "eio_linux" {= version & os = "linux"}
+  "eio_posix" {= version & os != "windows"}
```

Comment:
is there no `os_type` here like dune ?

Reply:
They're the ones from `opam var`, so it doesn't look like it.